README.md

```diff
@@ -100,6 +100,14 @@ yarn bundle:chrome
 yarn bundle:firefox
 ```
 
+### Release a new version
```

Comment:
Two tiny requests. In all the other sections we use:

1. Present progressive tense: Releasing a new version
2. A blank line after the heading

Can you update these? Just for consistency.

Thanks ✌️